app/pages/home/homePageController.ts

```diff
@@ -25,22 +28,26 @@ export class HomePageController {
   };
 
   public update() {
-    try {
-      this.dataService.list({
-        group: 'servicecatalog.k8s.io',
-        resource: 'serviceclasses'
-      }, {}, (resources: any) => {
-        this.ctrl.services = resources.by("metadata.name"); ;
-        this.ctrl.applications = this.constants.REDHAT_APPLICATIONS;
-        this.ctrl.categories = this.constants.SERVICE_CATALOG_CATEGORIES;
-        this.ctrl.loading = false;
-      });
-    } catch (e) {
-      console.log("Error Loading serviceclasses from Service Catalog.k8s.io: " + e);
-      this.ctrl.services = [];
-      this.ctrl.applications = this.constants.REDHAT_APPLICATIONS;
-      this.ctrl.categories = this.constants.SERVICE_CATALOG_CATEGORIES;
-      this.ctrl.loading = false;
-    }
+
+    // TODO: Remove these try/catches once we are not relying on 'additionalServers' in config.js
```

Comment:
since you switched to the promises can you remove this comment, since handling the promise rejection is a good thing to keep in case we ever get errors from the API or the server goes down